airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql

```diff
@@ -58,7 +57,7 @@ WITH
 SELECT
   calitp_itp_id,
   calitp_url_number,
-  textPayload,
+  token_cleaned_textpayload,
```

Comment:
Let's try to be consistent with what kind of case is being used. In this instance, I would expect something like:

```suggestion
  token_cleaned_text_payload,
```